src/trackinfoobject.cpp

```diff
@@ -783,13 +783,14 @@ bool TrackInfoObject::isDirty() {
 
 int TrackInfoObject::getRating() const {
     QMutexLocker lock(&m_qMutex);
-    return m_iRating;
+    return m_metadata.getRating();
```

Comment:
Mixxx rating can be != Traktor rating for some users or files. 
We have to avoid that we loose the Mixxx rating when we reload the meta data form the file.